README.md

```diff
@@ -35,9 +33,9 @@ It is dependent to jQuery and Lodash well as jsPsych.
 3. Create a question group with its welcome and ending screens.
 4. Create a question within the group.
 5. Question should be **Mandatory**, with _type_ set to **Short Free Text**.
-6. Within the _question_ field, use the WYSIWYG to create a single-row, double-column table.
-7. Fill the left column with what you'd want to see on the left half of the screen, and the right column with what you expect on the right half of the screen.
-8. Below the table, add the stimulus word for this block.
+6. Within the _question_ field, write the name of the concept that will be displayed on the left. Set its style to a title (any level). On a line below write a coma-separated list of concepts that are associated with this concept. Then below add again a title-styled concept followed by its coma-separated list of related words.
```

Comment:
@tsatse Could you take a screenshot of what you just explained, within your admin panel?